.gitlab-ci.yml

```diff
@@ -338,25 +338,25 @@ test:acceptance:
     - pip3 install -r requirements.txt
     - python3 -m pytest -v
       --mender-client-version $MENDER_VERSION
-      --mender-client-deb-version $(cat ${CI_PROJECT_DIR}/output/opensource/mender-client-deb-version)
+      --mender-client-deb-version $(cat ${CI_PROJECT_DIR}/output/opensource/debian-buster-amd64/mender-client-deb-version)
```

Comment:
S:
Why not make a variable out of this while we're at it?

Reply:
fixed